packages/node_modules/@ciscospark/react-component-activity-menu/src/index.test.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
+
+import ActivityMenu from '.';
+
+describe(`ActivityMenu component`, () => {
+  const onExit = jest.fn();
+  const onMeet = jest.fn();
+  const onMessage = jest.fn();
+  const component = renderer.create(
+    <ActivityMenu
+      onExit={onExit}
+      onMeet={onMeet}
+      onMessage={onMessage}
+    />
+  );
+
+  it(`renders properly`, () => {
+    expect(component).toMatchSnapshot();
+  });
```

Comment:
Yea, I think we would benefit from having the enzyme tests in anything that had click handlers and events.